cdap-docs/examples-manual/source/examples/log-analysis.rst

```diff
@@ -0,0 +1,222 @@
+- From the Standalone CDAP SDK directory, use the CDAP Command Line Interface::
+
+    $ cdap-cli.sh start service LogAnalysisApp.ResponseCounterService
+    $ cdap-cli.sh start service LogAnalysisApp.HitCounterService
+    $ cdap-cli.sh start service LogAnalysisApp.RequestCounterService
+    
+    Successfully started service 'ResponseCounterService' of application 'LogAnalysisApp' with stored runtime arguments '{}'
+    Successfully started service 'HitCounterService' of application 'LogAnalysisApp' with stored runtime arguments '{}'
+    Successfully started service 'RequestCounterService' of application 'LogAnalysisApp' with stored runtime arguments '{}'
+
+Injecting Access Logs
+---------------------
+
+Inject a file of Apache access log to the stream *logStream* by running this command from the
+Standalone CDAP SDK directory, using the Command Line Interface::
+  
+  $ cdap-cli.sh load stream logStream cdap-examples/LogAnalysis/resources/apache.accesslog "text/plain"
+  Successfully sent stream event to stream 'logStream'
+
+Running the Workflow
+--------------------
+There are three ways to start the workflow:
+
+1. Go to the *LogAnalysisApp* `application overview page
+   <http://localhost:9999/ns/default/apps/LogAnalysisApp/overview/status>`__,
+   click ``LogAnalysisWorkflow`` to get to the Workflow detail page, then click the *Start* button; or
+   
+#. Send a query via an HTTP request using the ``curl`` command::
+
+    $ curl -w'\n' -v \
+        http://localhost:10000/v3/namespaces/default/apps/LogAnalysisApp/workflows/LogAnalysisWorkflow/start
+
+#. Use the Command Line Interface::
+
+    $ cdap-cli.sh start workflow LogAnalysisApp.LogAnalysisWorkflow
+
+Querying the Results
+--------------------
+
+To query the *hitCount* KeyValueTable through the ``HitCounterService``, send a query via an HTTP
+request using the ``curl`` command. For example::
+
+  $ curl -w'\n' -X POST -d'{"url":"/index.html"}' http://localhost:10000/v3/namespaces/default/apps/LogAnalysisApp/services/HitCounterService/methods/hitcount
+
+You can also use the Command Line Interface::
+
+  $ cdap-cli.sh call service LogAnalysisApp.HitCounterService POST 'hitcount' body '{"url":"/index.html"}'
+
+On success, the above command will return the hit count for the above URL, such as::
+
+  4
+
+Similarly, to query the *responseCount* ``KeyValueTable`` through the *ResponseCounterService*, the *reqCount*
+``TimePartitionedFileSet`` through the *RequestCounterService*, and to retrieve data from a particular partition of
+the ``TimePartitionedFileSet``, use:
+
+curl and Command Line Interface::
+
+  $ curl -w'\n' http://localhost:10000/v3/namespaces/default/apps/LogAnalysisApp/services/ResponseCounterService/methods/rescount/200
+  $ cdap-cli.sh call service LogAnalysisApp.ResponseCounterService GET 'rescount/200'
+
+On success, the above command will return the total number of responses sent with the queried response code::
```

Comment:
On success, this command will return the total number of responses sent with the queried response code, ``30``.